typescript/sdk/src/utils/fork.ts

```diff
@@ -19,10 +14,10 @@ export enum ANVIL_RPC_METHODS {
 }
 
 /**
- * Resets the local node to it's original start (anvil [31337] at block zero).
+ * Resets the local node to it's original state (anvil [31337] at block zero).
  */
 export const resetFork = async () => {
-  logGray(`Resetting forked network...`);
+  rootLogger.info(`Resetting forked network...`);
```

Comment:
I recommend making a child logger for this file. `rootLogger.child({module:'fork-utils'})`